src/factories/nodeFlowRunState.ts

```diff
@@ -146,18 +147,32 @@ export async function nodeFlowRunStateFactory(state: RunGraphStateEvent) {
   function getRenderStyles(): StateRectangleRenderProps {
     const { background = '#fff' } = config.styles.state(state)
 
-    let startX = scale(state.timestamp) - scale(parentStart!)
+    const parentStartX = scale(parentStart!)
```

Comment:
I feel like this `!` could produce runtime errors. Could we do something type safe here?